src/app/tests/suites/certification/Test_TC_CDOCONC_1_1.yaml

```diff
@@ -0,0 +1,202 @@
+# limitations under the License.
+
+name: 145.1.1. [TC-CDOCONC-1.1] Global Attributes with DUT as Server
+
+PICS:
+    - CDOCONC.S
+
+config:
+    nodeId: 0x12344321
+    cluster: "Carbon Dioxide Concentration Measurement"
+    endpoint: 1
+
+tests:
+    - label: "Wait for the commissioned device to be retrieved"
+      cluster: "DelayCommands"
+      command: "WaitForCommissionee"
+      arguments:
+          values:
+              - name: "nodeId"
+                value: nodeId
+
+    - label: "Read the global attribute: ClusterRevision"
+      command: "readAttribute"
+      attribute: "ClusterRevision"
+      PICS: CDOCONC.S.Afffd
+      response:
+          value: 1
+          constraints:
+              type: int16u
+
+    - label: "Read the global attribute: FeatureMap"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS:
+          CDOCONC.S.Afffc && !CDOCONC.S.F00 && !CDOCONC.S.F01 && !CDOCONC.S.F02
+          && !CDOCONC.S.F03 && !CDOCONC.S.F04 && !CDOCONC.S.F05
+      response:
+          value: 0
+          constraints:
+              type: bitmap32
+
+    - label:
+          "Given CDOCONC.S.F00(MEA) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F00
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x1]
+
+    - label:
+          "Given CDOCONC.S.F01(LEV) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F01
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x2]
+
+    - label:
+          "Given CDOCONC.S.F02(MED) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F02
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x4]
+
+    - label:
+          "Given CDOCONC.S.F03(CRI) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F03
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x8]
+
+    - label:
+          "Given CDOCONC.S.F04(PEA) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F04
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x10]
+
+    - label:
+          "Given CDOCONC.S.F05(AVG) ensure featuremap has the correct bit set"
+      command: "readAttribute"
+      attribute: "FeatureMap"
+      PICS: CDOCONC.S.Afffc && CDOCONC.S.F05
+      response:
+          constraints:
+              type: bitmap32
+              hasMasksSet: [0x20]
+
```

Comment:
The test plan has "At least one of bit 0 and bit 1 SHALL be 1."  I don't see that reflected in this YAML.

Reply:
I didn't know how to do `featureMap & 0x3 != 0` in the YAML so I worked out the valid combinations of feature map based on the rules in the test plan and did an `anyOf` constraint.

`anyOf: [1, 2, 3, 6, 7, 10, 11, 14, 15, 17, 19, 23, 27, 31, 33, 35, 39, 43, 47, 49, 51, 55, 59, 63]`

Reply:
Spoke to vivien - raised issue #27248 and added a version of this test that works with C++ for now